doc/classes/GeometryInstance.xml

```diff
@@ -46,22 +46,26 @@
 		</member>
 		<member name="lod_max_distance" type="float" setter="set_lod_max_distance" getter="get_lod_max_distance" default="0.0">
 			The GeometryInstance's max LOD distance.
+			[b]Note:[/b] This property currently has no effect.
 		</member>
 		<member name="lod_max_hysteresis" type="float" setter="set_lod_max_hysteresis" getter="get_lod_max_hysteresis" default="0.0">
 			The GeometryInstance's max LOD margin.
+			[b]Note:[/b] This property currently has no effect.
 		</member>
 		<member name="lod_min_distance" type="float" setter="set_lod_min_distance" getter="get_lod_min_distance" default="0.0">
 			The GeometryInstance's min LOD distance.
+			[b]Note:[/b] This property currently has no effect.
 		</member>
 		<member name="lod_min_hysteresis" type="float" setter="set_lod_min_hysteresis" getter="get_lod_min_hysteresis" default="0.0">
 			The GeometryInstance's min LOD margin.
+			[b]Note:[/b] This property currently has no effect.
 		</member>
 		<member name="material_override" type="Material" setter="set_material_override" getter="get_material_override">
 			The material override for the whole geometry.
-			If there is a material in [code]material_override[/code], it will be used instead of any material set in any material slot of the mesh.
+			If there is a material in [member material_override], it will be used instead of any material set in any material slot of the mesh.
```

Comment:
Probably not really useful to repeat the property name in its own description, maybe "If a material is assigned to this property, ..."